src/Utils/utils.ts

```diff
@@ -571,3 +571,14 @@ export const copyToClipboard = async (content: string) => {
     Notification.Error({ msg: "Copying is not allowed" });
   }
 };
+
+export const calculateDateRangeDuration = (
```

Comment:
duration can be expressed in many ways. since this returns in days, use proper namings.

```suggestion
export const calculateDaysBetween = (
```